backend/app/src/main/resources/templates/admin/profanity-words.html

```
@@ -0,0 +1,80 @@
+<main>
+    <a href="/admin" class="nav-back">← 대시보드로</a>
+
+    <div class="section-header" style="margin-top: 1.5rem;">
+        <span class="section-title">단어 추가</span>
+    </div>
+
+    <form th:action="@{/admin/profanity/words}" method="post" style="margin-bottom: 2rem;">
+        <div style="display:flex; gap:0.75rem; align-items:flex-end;">
+            <div>
+                <label style="display:block; font-size:0.85rem; margin-bottom:0.25rem;">단어</label>
+                <input type="text" name="word" required maxlength="200"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)